app/process/watcher/transfer/watcher.go

```diff
@@ -167,6 +172,7 @@ func (ctw Watcher) processTransaction(tx model.Transaction, q qi.Queue) {
 		ctw.logger.Errorf("[%s] - Sanity check failed. Error: [%s]", tx.TransactionID, err)
 		return
 	}
+	ctw.initializeSuccessRatePrometheusMetrics(tx, constants.HederaNetworkId, targetChainId, asset)
```

Comment:
This can be done a few lines below. If someone bridges an asset we do not support, we will create the metrics regardless